docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:15-alpine
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+
+  celery:
```

Comment:
Can we add web as a dependency then use that image to launch celery ? That way we don't need to have separate Dockerfile for celery

Reply:
I didn't get this, can you elaborate? or may be take it up separately as an issue?